### ISSUE TITLE
Custom button is not getting activated using screenreader JAWS

### ISSUE THREAD

**Issue by JulesAchiel** [ckeditor/ckeditor-dev#883]
## Are you reporting a feature request or a bug?
Bug

## [Check if the issue is already reported](http://docs.ckeditor.com/#!/guide/dev_issues_readme-section-avoid-duplicates)
none found

## Provide detailed reproduction steps (if any)
1. open up jsfiddle https://jsfiddle.net/JulesAchiel/Lj9kmu6b/
2. Standard CKEditor 4.6.2 is loaded
3. a custom button 'Insert Knowledge' is added to the toolbar
4. Start screenreader JAWS
5. put focus on the pane with CKEditor (not in CKEditor itself)
6. key combination CTRL+INSERT+F will open a search screen from JAWS to search for an item
7. search for 'Insert Knowledge'
8. only one item should get read out by JAWS, but in case it is not reading out the button, then use F3 to go to the next item.
9. once JAWS read out 'Insert Knowledge button' the user expects to press ENTER to activate the button which is not happening.

10. There is a standard HTML button outside CKEditor 'Click Me'
11. key combination CTRL+INSERT+F will open a search screen from JAWS to search for an item
12. search for 'click Me'
13. once JAWS reads out 'Click Me button' press ENTER and an alert is shown as the button has been activated by JAWS.

### Expected result

JAWS user should be able to activate the custom button in CKEditor

### Actual result

JAWS user is unable to activate the custom button by searching for the button label and pressing ENTER to activate

## Other details

* Browser: all browsers
* OS: N/A
* CKEditor version: 4.6.2
* Installed CKEditor plugins: none

**Comment by wojtekw92** [comment]
Hi @JulesAchiel 
Can you update your CKEditor to the last stable release(4.7.2) and check if issue occurs?

**Comment by JulesAchiel** [comment]
I have updated the JSFIDDLE to use CKEditor 4.7.2 and the problem still reproduces

**Comment by jswiderski** [comment]
It seems you are able to find a button or link outside of editor and invoke its onclick event handler with Enter. We are also able to find editor button like `Source` (Źródło dokumentu) but whenever you press enter, focus goes into editor and event handler doesn't get called.

**Comment by JulesAchiel** [comment]
Do we have an ETA of when this issue will be fixed?

**Comment by JulesAchiel** [comment]
When can we expect this to be uptaken for a fix?

**Comment by mlewand** [comment]
After a further research, this turns to be just how JAWS treats buttons in a `[role=application]` element.

Check:

* [Codepen with buttons in a application element](https://codepen.io/mlewand/pen/Yaybdq)
* [Codepen with buttons with **no** application element](https://codepen.io/mlewand/pen/Gxpbxx)

Thus this is a JAWS issue, where JAWS team should fix so that buttons in a `[role=application]` element gets focused properly upon matching using find function.

It's worth to mention that with current JAWS implementation pressing "enter" causes the button to become focused, where pressing "enter" again triggers the button as expected.
